### DISCUSSION TITLE
gap in tailwindcss

### DISCUSSION THREAD

**Discussion by Long-le-mt** [tailwindlabs/tailwindcss#11375] [Help]
I use tailwindcss with nextjs, when using gap with class gap-x-3 it doesn't work, with gap-x-1 gap-x-2 it works normally, some classes are lost when I use it. Although I checked these classes are still on the document so we won't be configuring the tailwind.config.js file.
I using tailwind ver 3.3.2

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- `3` is a valid value for `gap`.
